src/transformers/models/big_bird/modeling_big_bird.py

```diff
@@ -568,7 +568,12 @@ def bigbird_block_sparse_attention(
         if from_seq_len in [1024, 3072, 4096]:  # old plans used in paper
             rand_attn = [
                 self._bigbird_block_rand_mask(
-                    self.max_seqlen, self.max_seqlen, from_block_size, to_block_size, n_rand_blocks, last_idx=1024
+                    self.max_seqlen,
+                    self.max_seqlen,
+                    from_block_size,
+                    to_block_size,
+                    n_rand_blocks,
+                    last_idx=1024,
```

Comment:
There is no need for the restyle anymore here, might be some leftover from when there was `deterministic` passed?